packages/client/public/sw.js

```diff
@@ -0,0 +1,60 @@
+const CACHE_NAME = 'cats-paws-game-v1'
+
+const STATIC_ASSETS = ['/', '/index.html', '/FastPaws.svg', '/not-found']
+
+const getCacheUrls = () => {
+  try {
+    return JSON.parse('%HASHURLS%')
+  } catch (err) {
+    return []
+  }
```

Comment:
Можно не прописать err, если его не используешь 
```suggestion
  } catch {
    return []
  }
```